resources/metadata/kanikoExecute.yaml

```diff
@@ -144,6 +144,36 @@ spec:
         resourceRef:
           - name: commonPipelineEnvironment
             param: artifactVersion
+      - name: containerMultipleBuild
+        type: bool
+        description: Defines if multiple containers should be build using the same Dockerfile.
+        scope:
+          - GENERAL
+          - PARAMETERS
+          - STAGES
+          - STEPS
+      - name: containerImageNames
+        type: "[]string"
+        description: Names of the containers which will be built - using the same context (Dockerfile)
+        mandatoryIf:
+          - name: containerMultipleBuild
+            value: true
+        scope:
+          - GENERAL
+          - PARAMETERS
+          - STAGES
+          - STEPS
+      - name: containerImageTags
```

Comment:
We already have the `containerImageTag` parameter which in General Purpose Pipelines comes from the `artifactPrepareVersion` step via Common Pipeline Environment values. I think the creation of a new parameter is not needed here, we can use the existing parameter.